packages/neuron-wallet/src/keys/tool.ts

```diff
@@ -0,0 +1,83 @@
+import { BIP32 } from 'bip32'
+import { Child } from './keystore'
+
+// BIP44: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+const PURPOSE = "44'"
+// 360 is tentative value
+const COIN_TYPE_TESTNET: string = "360'"
+const ACCOUNT = "0'"
+// external address
+const CHANGE = 0
+
+export default class Tool {
+  private static root: BIP32
+
+  public static searchUsedChildKeys(root: BIP32) {
```

Comment:
`static` method cannot access to `this`, and this should be alerted by `eslint`

Reply:
Here `Tool` replace `this` ?

Reply:
if `root` is a property of `Tool` instance, `Tool` class cannot access to it. You can set this method to instance method(remove `static`), or pass the instance to it.

Reply:
root is not a instance of Tool.

Reply:
I mean, `this.root` is to get `root` property of `Tool` instance, and `Tool` static method cannot get the instance, namely `Tool` static method cannot get `this.root`

Reply:
OK. I see. But how can I get `root` from `Tool`? `Tool` is just a tool class, so I think it should be a static class.

Reply:
In this function you are going to update the `this.root`, so just pass `this`(namely the instance` in:
```
public static searchUsedChildKeys(tool: Tool, root: BIP32) {
  tool.root = root
}
```